packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -116,12 +117,29 @@ export class ActiveWorkflowRunner {
 			throw new ResponseHelper.ResponseError('The "activeWorkflows" instance did not get initialized yet.', 404, 404);
 		}
 
-		const webhook = await Db.collections.Webhook?.findOne({ webhookPath: path, method: httpMethod }) as IWebhookDb;
+		let webhook = await Db.collections.Webhook?.findOne({ webhookPath: path, method: httpMethod }) as IWebhookDb;
+		let webhookId: string | undefined;
 
-		// check if something exist
+		// check if path is dynamic
 		if (webhook === undefined) {
-			// The requested webhook is not registered
-			throw new ResponseHelper.ResponseError(`The requested webhook "${httpMethod} ${path}" is not registered.`, 404, 404);
+			// check if a dynamic webhook path exists
+			const pathElements = path.split('/');
+			webhookId = pathElements.shift();
+			webhook = await Db.collections.Webhook?.findOne({ webhookId, method: httpMethod }) as IWebhookDb;
+			if (webhook === undefined) {
+				// The requested webhook is not registered
+				throw new ResponseHelper.ResponseError(`The requested webhook "${httpMethod} ${path}" is not registered.`, 404, 404);
+			}
+			path = webhook.webhookPath;
+			// extracting params from path
+			const webhookPathParams: IDataObject = {};
+			webhook.webhookPath.split('/').forEach((ele, index) => {
+				if (ele.startsWith(':')) {
+					webhookPathParams[ele.slice(1)] = pathElements[index];
```

Comment:
Currently the webhook-id shows up as "0" under "params". That is probably more confusing than helpful and should so be removed.

Reply:
this is the default from express, it provides the whole path under the `0` key in `req.params` since our Webhook routing works with a `*` wildcard (see image below). I am adding the key-value pairs from the route. If you want we can also just return `webhookPathParams `, but then we would lose any data from express's `req.params`.

<img width="872" alt="Screen Shot 2021-01-16 at 14 27 56" src="https://user-images.githubusercontent.com/1849459/104813106-43c60b80-5807-11eb-807d-ef7c7e110b07.png">

Reply:
Not sure if I understand it correctly but there should probably never be anything in `req.params` by default that we need?

Reply:
yes indeed! changing it to just return an object with the params 👍🏼